src/libs/actions/Report.ts

```diff
@@ -2162,12 +2169,35 @@ function leaveRoom(reportID: string, isWorkspaceMemberLeavingWorkspaceRoom = fal
     };
 
     const parameters: LeaveRoomParameters = {
-        reportID,
+        reportID: currentReportID,
     };
 
     API.write('LeaveRoom', parameters, {optimisticData, successData, failureData});
 
-    if (isWorkspaceMemberLeavingWorkspaceRoom) {
+    const sortedReportsByLastRead = ReportUtils.sortReportsByLastRead(Object.values(allReports ?? {}) as Report[], reportMetadata);
+
+    // We want to filter out the current report, hidden reports and empty chats
+    const filteredReportsByLastRead = sortedReportsByLastRead.filter(
+        (report) =>
```

Comment:
wouldn't this be easier if we just rename this to `sortedReport` instead of renaming the above report variable?

Reply:
Ah, yes `sortedReport` is definitely better!

Reply:
Updated